crypto/encoding/codec.go

```diff
@@ -14,6 +14,7 @@ func init() {
 	json.RegisterType((*pc.PublicKey)(nil), "tendermint.crypto.PublicKey")
 	json.RegisterType((*pc.PublicKey_Ed25519)(nil), "tendermint.crypto.PublicKey_Ed25519")
 	json.RegisterType((*pc.PublicKey_Secp256K1)(nil), "tendermint.crypto.PublicKey_Secp256K1")
+	json.RegisterType((*pc.PublicKey_Secp256K1Uncompressed)(nil), "tendermint.crypto.PublicKey_Secp256K1Uncompressed")
```

Comment:
@sergio-mena @lasarojc in upstream will the protobuf types also be registered under `comet` namespace ? Anything else we should consider here as well ?

Reply:
One of the main changes in `v1.x` (which we are trying to release in Q1) is the heaving renaming and versioning of our protobufs. So, yes, there is a renaming from "tendermint" to "cometbft" coming up.

Reply:
So in this fork if we register them under `tendermint`, won't that be a problem when we move upstream ?

Reply:
I took a deeper look into this. Despite the heavy renaming we're doing in protobufs (`tendermint` --> `cometbft`), these names have stayed registered to `tendermint`, so this is not changing in `v1` (I don't think we have the time to squeeze such a change before `v1`). So these `RegisterType` calls will stay on Tendermint for at least one more version.

Reply:
Resolved (AFAIAC)

Reply:
Got it, thanks! Just for my understanding, and we discussed in the previous meeting, but even if the name changes it won't matter because the underlying type remains the same correct ?